django/thunderstore/community/templates/community/communities_list.html

```diff
@@ -0,0 +1,215 @@
+            <li class="nav-item">
+                <a href="{% url "settings.linked-accounts" %}" class="nav-link">Settings</a>
+            </li>
+            {% if request.user.is_staff %}
+            <li class="nav-item">
+                <a href="{% url "admin:index" %}" class="nav-link">Admin</a>
+            </li>
+            {% endif %}
+            <li class="nav-item">
+                <a href="{% url "logout" %}" class="nav-link">Logout</a>
+            </li>
+        {% else %}
+            <li class="nav-item">
+                <a href="{% add_auth_login_link "github" %}" class="nav-link"><span class="fab fa-github"></span> Log in with GitHub</a>
+            </li>
+            <li class="nav-item">
+                <a href="{% add_auth_login_link "discord" %}" class="nav-link"><span class="fab fa-discord"></span> Log in with Discord</a>
+            </li>
+        {% endif %}
+    </ul>
+</nav>
+</div>
+</div>
+{% endblock %}
+
+{% block content %}
+{% cache_until "any_package_updated" "mod-list" 1 page_obj.number cache_vary %}
+
+<div class="row mb-5 mt-5">
+    <div class="col-8">
+        <div class="row justify-content-left ml-0 align-text-bottom mod-count-box">
+            <img class="d-flex align-items-end" src="{% static 'icon.png' %}">
+            <p class="col-10 mt-4 d-flex align-items-end">Distributing <span class="ts-text-highlight pl-1 pr-1">{{ mod_count }}</span> mods, across <span class="ts-text-highlight pl-1 pr-1">{{ game_count }}</span> games</p>
+        </div>
+        <div class="row justify-content-left mb-10">
+            <h1 class="col-10 mt-4 d-flex align-items-end">Download and share mods with friends</h1>
+        </div>
+        <div class="row justify-content-left mb-3">
+            <p class="col-10 mt-4 text-left text-muted">Thunderstore hosts a massive collection of mods for <b>almost</b> every game on the market. Using mods is easy with our mod manager (see more below). For the developers we offer an <a href="{% url 'swagger' %}">API</a> and <a href="https://github.com/thunderstore-io/thunderstore-cli">CLI</a> tools that can be used to automate your workflow.</p>
+        </div>
+        <div class="row justify-content-left ml-0">
+            <a class="btn btn-primary mr-2 ml-0 ts-primary-button" href="#mod-manager">Mod manager</a>
+        </div>
+    </div>
+    <div class="col-4">
+        <img class="modspic float-right" src="{% static 'Modspic.png' %}">
+    </div>
+</div>
+<div class="flex-row mt-5">
+    <div class="row mb-2 ml-2 mr-2">
+        <div class="col-8">
+            <div class="row justify-content-left">
+                <h3 class="mb-1" >Find your game</h3>
+                <p class="align-items-end text-dark mt-2 ml-4" >Missing a game you'd like to see? Message us on <a href="https://discord.gg/UWpWhjZken">Discord</a>.</p>
+            </div>
+        </div>
+    </div>
+
+    {% if selectable_community_sites %}
+        <div class="d-flex flex-row flex-wrap package-list">
```

Comment:
Nitpicking, but this isn't a `package-list`, so the class name is misleading. Not worth to deal with if it's used in bazillion places I think.

Reply:
Actually the `package-list` only game some margins that are not needed. Not sure why it was used by me originally. Probably a copypasta